src/main/java/fi/otavanopisto/kuntaapi/server/controllers/ServiceController.java

```diff
@@ -86,9 +97,12 @@ public Service findService(ServiceId serviceId) {
       }
       
       return new SearchResult<>(result, searchResult.getTotalHits());
+    } else {
+//      List<Service> services = listServices(organizationId, firstResult, maxResults);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)